src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -216,11 +226,25 @@ public function getContentData(PropertyInterface $property)
         $filters = $property->getValue();
         $filters['excluded'] = [$property->getStructure()->getUuid()];
 
+        // default value of tags is an empty array
+        if (!array_key_exists('tags', $filters)) {
+            $filters['tags'] = [];
+        }
+
+        // extends selected filter with requested tags
+        $filters['websiteTags'] = $this->tagRequestHandler->getTags($params['tags_parameter']->getValue());
+        $filters['websiteTagOperator'] = $params['website_tag_operator']->getValue();
+
         // resolve tags to id
-        if (!empty($filters['tags'])) {
+        if (!empty($filters['tags']) && count($filters['tags']) > 0) {
```

Comment:
Is this addition really necessary? If the count of an array is 0, empty should also return false.